drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -4463,8 +4463,9 @@ void RasterizerStorageGLES3::mesh_render_blend_shapes(Surface *s, const float *p
 	for (int ti = 0; ti < mtc; ti++) {
 		float weight = p_weights[ti];
 
-		if (weight < 0.00001) //not bother with this one
+		if (weight <= 0.00001 && weight >= -0.00001) {
```

Comment:
This order is more natural to read:

```suggestion
		if (weight >= -0.00001 && weight <= 0.00001) {
```

Reply:
Such timing XD

Reply:
hmm, let me check because I think this relates more to the steps in the editor rather than that check.

Reply:
Nope, it's the editor steps, not the is equal to zero.

Reply:
is_zero_approx visual artefacts

Reply:
Hmm, now I'm getting some minor flickering on unrelated vertexes seems this check is kinda flaky.

Reply:
OK so rebooting fixed my artefacts and reimporting worked. I think this change will need to be tested a bit more.

Reply:
I have a driver problem with blend shapes on 3.2 and mesa-git I am testing on OSX now....